modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/core/thunks/fieldChange.es.js

```diff
@@ -12,8 +12,11 @@ let REVALIDATE_UPDATES = [];
 
 const skipPageEvaluationFieldNames = ['name', 'requiredErrorMessage'];
 
-const needsPageEvaluation = (fieldName) => {
-	return !skipPageEvaluationFieldNames.includes(fieldName);
+const needsPageEvaluation = (fieldName, fieldInstance) => {
+	return (
+		!skipPageEvaluationFieldNames.includes(fieldName) &&
+		!fieldInstance.name.includes('ObjectDefinitionsPortlet')
```

Comment:
If someone changes the value of the name property, we will have a problem here. I think we can do this through the BE.

In `ObjectFieldBusinessType` we can pass the evaluable value in the `getProperties` function.

I believe that the `DDMFormFieldTemplateContextFactory` will be responsible for getting this property and setting it to the corresponding value here:

```
		_setDDMFormFieldTemplateContextEvaluable(
			ddmFormFieldTemplateContext, changedProperties,
			ddmFormField.getProperty("evaluable"));
```

But @marcelabc can confirm if it is a better approach